Content.Shared/Interaction/RotateToFaceSystem.cs

```diff
@@ -80,14 +80,8 @@ public bool TryFaceCoordinates(EntityUid user, Vector2 coordinates, TransformCom
 
         public bool TryFaceAngle(EntityUid user, Angle diffAngle, TransformComponent? xform = null)
         {
-            if (_actionBlockerSystem.CanChangeDirection(user))
-            {
-                if (!Resolve(user, ref xform))
-                    return false;
-
-                _transform.SetWorldRotation(xform, diffAngle);
-                return true;
-            }
+            if (!_actionBlockerSystem.CanChangeDirection(user))
+                return false;
```

Comment:
Doesn't this break turrets or strafing?

Reply:
No, it doesn't (I tested walking around while in harm, and hostile turrets sure do still tear my poor character to shreds)

This change, combined with the change in SharedBuckleSystem, maintain the prior code's behavior except where an entity that Can(not)ChangeDirection is buckled onto another. Neither turrets nor strafing involve buckling.